### ISSUE TITLE
ECS services never seem to transition into running state

### ISSUE THREAD

**Issue by davido-nw** [getmoto/moto#6283]
Using Moto 4.1.8. When creating a service in a cluster, I can't seem to figure out how, or if, there's any way to get the services to transition to a running state. I have the following code for fixtures:
```python
@pytest.fixture
def ecs(credentials):
    with moto.mock_ecs():
        yield boto3.client("ecs")

@pytest.fixture
def setup_ecs(ecs):
    ecs.create_cluster(clusterName="derp")
    ecs.register_task_definition(
        family="derp",
        containerDefinitions=[
            {
                "name": "task",
                "image": "123456789012.dkr.ecr.us-west-2.amazonaws.com/derp:live",
                "cpu": 1024,
                "memory": 2048,
                "memoryReservation": 2048,
                "portMappings": [
                    {"containerPort": 8000, "hostPort": 8000, "protocol": "tcp"}
                ],
                "essential": True,
                "mountPoints": [],
                "volumesFrom": [],
                "linuxParameters": {"initProcessEnabled": True},
                "logConfiguration": {"logDriver": "json-file"},
            }
        ],
    )
    ecs.create_service(
        cluster="derp",
        serviceName="derp",
        taskDefinition="derp",
        desiredCount=1,
    )
```
When I introspect the `derp` service, I notice that the service itself has a desired/pending/running count of 1/0/0, and when I introspect the `deployments` specifically, I have a 1/1/0 count of desired/pending/running.

What I'm trying to test is the scaling down and scaling up of the service count, in which I'm hoping to use `runningCount` as a way of knowing whether the service has scaled down to 0 or not. The source code of the tests doesn't seem to handle the scenario where a service transitions to a runningCount.

Is there a way to make the transition to `running` happen?

**Comment by rafcio19** [comment]
@davido-nw with ☝️ PR you will be able to use a new env var `MOTO_ECS_SERVICE_RUNNING` to set a number of running tasks, after the service is created. If you want to test the `runningCount` going up or down, then you will need to flip that env var accordingly. 

The PR will need to be reviewed by a maintainer but the env var approach is used in other parts of moto, so it fits the commonly adopted pattern.

**Comment by RajasGujarathi** [comment]
This behaviour is not described in the documents. Or I am missing something here 🤔

**Comment by bblommers** [comment]
Good catch @RajasGujarathi - I've added it to the ECS documentation here: https://docs.getmoto.org/en/latest/docs/services/ecs.html
